sdk/main/src/AccountCreateTransaction.cc

```diff
@@ -19,233 +19,156 @@
  */
 #include "AccountCreateTransaction.h"
 
+#include "PublicKey.h"
+#include "TransactionResponse.h"
+
 #include "helper/DurationConverter.h"
 
 #include <proto/crypto_create.pb.h>
-#include <proto/schedulable_transaction_body.pb.h>
-
-#include "PublicKey.h"
+#include <proto/response.pb.h>
+#include <proto/transaction.pb.h>
 
 namespace Hedera
 {
 //-----
 AccountCreateTransaction::AccountCreateTransaction()
-  : Transaction()
-  , mKey()
-  , mAccountMemo()
-  , mInitialBalance()
-  , mReceiverSignatureRequired(false)
-  , mAutoRenewPeriod(90)
-  , mMaxAutomaticTokenAssociations(0)
-  , mStakedAccountId()
-  , mStakedNodeId()
-  , mDeclineStakingReward(false)
-  , mAlias()
-{
-  mDefaultMaxTransactionFee = Hbar::from(5LL);
-}
-
-//-----
-AccountCreateTransaction::AccountCreateTransaction(
-  const std::unordered_map<TransactionId, std::unordered_map<AccountId, proto::TransactionBody>>& transactions)
-  : Transaction(transactions)
-{
-  initFromTransactionBody();
-}
-
-//-----
-AccountCreateTransaction::AccountCreateTransaction(const proto::TransactionBody& transaction)
-  : Transaction(transaction)
-{
-  initFromTransactionBody();
-}
-
-//-----
-void AccountCreateTransaction::validateChecksums(const Client& client) const
-{
-  if (mStakedAccountId.isValid())
-  {
-    mStakedAccountId.getValue().validateChecksum(client);
-  }
-}
-
-//-----
-void AccountCreateTransaction::onFreeze(proto::TransactionBody* body) const
 {
-  body->set_allocated_cryptocreateaccount(build());
+  mMaxTransactionFee = Hbar::from(5LL);
```

Comment:
Is this literal `5` documented anywhere?

Reply:
https://docs.hedera.com/guides/mainnet/fees#transaction-and-query-fees